schema/vrs.yaml

```diff
@@ -396,27 +387,77 @@ definitions:
     required: [ "type", "gene_id" ]
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Range
+  # Numerics, Comparators, and Ranges
+
+  IntegerOrRange:
+    description: >-
+      A set of classes that are used to represent either single integer values
+      (such as SimpleInteger) or integer ranges (such as IntegerRange).
+    oneOf:
+      - $ref: "#/definitions/IntegerValue"
+      - $ref: "#/definitions/IntegerRange"
+    discriminator:
+      propertyName: type
 
-  Range:
+  IntegerValue:
     description: >-
-      An inclusive range of integers. A range may be unbounded by omitting either
-      min or max values. Fully unbounded ranges are not permitted (one of min or
-      max MUST be specified).
+      An abstract class for related integer-based value classes.
     type: object
+    oneOf:
+      - $ref: "#/definitions/SimpleInteger"
+      - $ref: "#/definitions/OpenInteger"
+    discriminator:
+      propertyName: type
+
+  SimpleInteger:
```

Comment:
Can we please drop the Simple part? Everytime I read "SimpleInteger" I wanted to put a bag over my head. "Integer" should suffice.

Reply:
Will implement.